src/backend/distributed/planner/modify_planner.c

```diff
@@ -102,7 +102,6 @@ ErrorIfQueryNotSupported(Query *queryTree)
 	Oid distributedTableId = ExtractFirstDistributedTableId(queryTree);
 	uint32 rangeTableId = 1;
 	Var *partitionColumn = PartitionColumn(distributedTableId, rangeTableId);
-	char partitionMethod = PartitionMethod(distributedTableId);
```

Comment:
It's addressed by https://github.com/citusdata/citus/pull/450/files so better to remove it from this change.